plugins/@fingerprint/kvStorageUnsealedResult/index.ts

```diff
@@ -0,0 +1,14 @@
+import { KVStore } from 'fastly:kv-store'
+import { PluginType, registerPlugin } from '../../../src/utils/registerPlugin'
+import { EventResponse } from '@fingerprintjs/fingerprintjs-pro-server-api'
+
+async function fingerprintProcessUnsealedResult(requestId: string, data: EventResponse | null) {
+  const store = new KVStore('FingerprintResults')
+  await store.put(requestId, JSON.stringify(data))
```

Comment:
We could give an option to specify the store name

Reply:
This is out of our scope because it's a plugin and shouldn't use anything inside src folder as much as possible. Also whole plugin is commented out, if the customer needs to enable it they should uncomment it first.